test/twitter.test.js

```diff
@@ -218,7 +218,7 @@ describe('posting', () => {
 
 describe('uploading', () => {
   let uploadClient;
-  beforeAll(() => (uploadClient = newClient('upload')));
```

Comment:
@rlueder edited the upload test suite, so users don't have to explicitly edit the subdomain on calls to the media endpoints.